src/constants.py

```diff
@@ -31,6 +33,7 @@
 DB_RELATION_NAME = "pg-database"
 HYDRA_RELATION_NAME = "hydra-endpoint-info"
 LOGIN_UI_RELATION_NAME = "ui-endpoint-info"
+KRATOS_INFO_RELATION_NAME = RELATION_NAME
```

Comment:
Let's hard code the relation name here to keep it consistent. This name comes from charmcraft.yaml, not the relation lib